tests/integration/conftest.py

```diff
@@ -475,7 +469,7 @@ async def grafana_fixture(
     model: Model, application_name: str
 ) -> typing.AsyncGenerator[Application, None]:
     """Deploy and return grafana charm application with relation to WordPress charm."""
-    grafana = await model.deploy("grafana-k8s", channel="stable", trust=True)
+    grafana = await model.deploy("grafana-k8s", channel="latest/edge", trust=True)
```

Comment:
Same as above.

Reply:
Added comments for all the `edge` deployments. Thanks for the suggestion!